src/components/SearchBarPopup/searchBarPopup.scss

```diff
@@ -0,0 +1,129 @@
+@import "../variables.scss";
```

Comment:
The search results look for the search bar popup is actually different from the header search bar. The whole popup should contain the search results (since it's a little ugly to have a search results popup over another popup)
 
![image](https://user-images.githubusercontent.com/33335963/84990259-bc41de00-b0f9-11ea-848f-1266602b09fd.png)